.github/workflows/build-test-release.yml

```diff
@@ -202,6 +202,9 @@ jobs:
           name: version
           path: .
 
+      - uses: ./.github/actions/enable-msvc
+        if: runner.os == 'Windows'
```

Comment:
for consistency
```suggestion
        if: runner.os == 'windows'
```

Reply:
It has to be big letter. This is not `matrix.os`. `runner.os` is a different GA variable.